platform/files/handle_toml.go

```diff
@@ -0,0 +1,197 @@
+// Package files contains schema and helper methods for working with lifecycle configuration files.
+package files
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/buildpacks/lifecycle/api"
+	"github.com/buildpacks/lifecycle/buildpack"
+	"github.com/buildpacks/lifecycle/internal/encoding"
+	"github.com/buildpacks/lifecycle/log"
+)
+
+// Handler is the default handler used to read and write lifecycle configuration files.
+var Handler = &TOMLHandler{}
+
+// TOMLHandler reads and writes lifecycle configuration files in TOML format.
+type TOMLHandler struct{}
```

Comment:
If we ever wanted the platform to be able to supple JSON or YAML files, we could implement an alternate handler with minimal other changes.